### PR TITLE
Bug/d3 asim 2963

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

**Review by faizan2590 — APPROVED** [review]
LGTM

File: src/d3a/d3a_core/sim_results/__init__.py [hunk 1]
```diff
@@ -1,28 +1,30 @@
 from d3a.d3a_core.util import area_name_from_area_or_iaa_name
 
 
-def _is_cell_tower_type(area):
+def is_cell_tower_type(area):
```
Comment (spyrostz) [inline src/d3a/d3a_core/sim_results/__init__.py:4]: Thanks ! These should be public now indeed.

File: src/d3a/d3a_core/sim_results/device_statistics.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 from typing import Dict
 from d3a import limit_float_precision
 from d3a_interface.utils import create_or_update_subdict
-
+from d3a.d3a_core.sim_results import is_load_node_type, is_cell_tower_type, is_pv_node_type, \
+    is_prosumer_node_type
 FILL_VALUE = None
```
Comment (spyrostz) [inline src/d3a/d3a_core/sim_results/device_statistics.py:6]: Trivial, but I am wondering why flake does not complain. I was under the impression one empty line was needed here. Nevermind, if flake does not complain all is well.

**Review by spyrostz — APPROVED** [review]
LGTM

File: src/d3a/d3a_core/sim_results/device_statistics.py (hunk 2 above)
Comment (hannesdiedrich) [reply src/d3a/d3a_core/sim_results/device_statistics.py:6]: No, it didn't weirdly.

**Pull request merged**